third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1117,9 +1151,32 @@ impl Interpreter {
     fn get_internal_state(&self) -> ExecutionState {
         self.get_stack_frames(usize::MAX)
     }
+}
+
+impl InterpreterDebugInterface for InterpreterImpl {
+    #[allow(dead_code)]
```

Comment:
Can we use some `cfg(any(...))` or so? Seems like this is only used if some cfg feature is on

Reply:
Inherited so far, will put on my todo to update